Sources/SwiftProtobuf/BinaryDecodingOptions.swift

```diff
@@ -13,7 +13,7 @@
 // -----------------------------------------------------------------------------
 
 /// Options for binary decoding.
-public struct BinaryDecodingOptions {
+public struct BinaryDecodingOptions: Sendable {
```

Comment:
I believe this needed to be [`_ProtoSendable`](https://github.com/apple/swift-protobuf/blob/main/Sources/SwiftProtobuf/ProtoSendable.swift) and not `Sendable`, because we still support Swift < 5.5, so `Sendable` doesn't always exist.

Reply:
This is on `main`. What is our Swift policy for releasing the next `main` version?

Reply:
Hmmm...  I thought we had discussed changing the policy to "1 year old Swift releases" rather than the previous "1 major version".  (The two used to be nearly synonymous, but no longer are.)

Reply:
FWIW I would recommend the NIO way which is the last three Swift releases. However, if we do a new major we would also consider only supporting the latest minor and build up from there if there are features that we could otherwise not use.

Reply:
We updated `main` to 5.0 - https://github.com/apple/swift-protobuf/blob/main/README.md#system-requirements

Doesn't look like we had much discussion in https://github.com/apple/swift-protobuf/issues/1210, what I can find in email didn't really talk about linking it to any date thing.

Swift Package Index seems to be currently showing the computed data for 4 version (5.9 and back), so that would also sorta slight with the last three minor Swift releases.

I'm not opposed to doing that, but with 6.x on the horizon I do wonder if by the time we see 6.2 would it be ok to drop 5.9 at that point, since we don't know how much change 6.x will be; but maybe that just means we just move the min language version up to 5.6 for now, and not officially state it will always be 3 releases?

Also, do we want to move the 1.x branch up?